apps/s_client.c

```diff
@@ -660,9 +660,9 @@ const OPTIONS s_client_options[] = {
 #ifndef OPENSSL_NO_OCSP
     {"status", OPT_STATUS, '-', "Request certificate status from server"},
     {"ocsp_check", OPT_STATUS_OCSP_CHECK, '-',
-     "check leaf certificate revocation with OCSP response"},
+     "require leaf certificate status checking, trying OCSP stapling first"},
     {"ocsp_check_all", OPT_STATUS_OCSP_CHECK_ALL, '-',
-     "check full chain revocation with OCSP response"},
+     "require certificate status checking on each cert in chain, trying OCSP stapling first"},
```

Comment:
The new string unfortunately is too long for a single output line.
Please shorten or split using 
```
{OPT_MORE_STR, 0, 0, "<2nd line>"},
```